src/Acuminator/Acuminator.Analyzers/StaticAnalysis/DacPropertyAttributes/IncompatibleDacPropertyAndFieldAttributeFix.cs

```diff
@@ -64,59 +65,51 @@ private void RegisterCodeFixForPropertyType(SyntaxNode root, SyntaxNode codeFixN
 
 		private void RegisterCodeFix(SyntaxNode root, AttributeSyntax attributeNode, CodeFixContext context)
 		{
+			PropertyDeclarationSyntax propertyNode = attributeNode.Parent<PropertyDeclarationSyntax>();
+
+			if (propertyNode == null || context.CancellationToken.IsCancellationRequested)
```

Comment:
`ThrowIfCancellationRequested`?

Reply:
@dropsonic this is a code executed before registration of the code fix. Should we throw here? Will it be handled correctly?

Reply:
@SENya1990 as far as I know, yes, because MS examples use a lot of Roslyn methods which throw instead of safe checking.

Reply:
ok, I'll fix it